pennylane/transforms/sign_expand/sign_expand.py

```diff
@@ -324,13 +348,12 @@ def processing_fn(res):
     # make one tape per observable
     tapes = []
     for proj in projs:
-        with tape.__class__() as new_tape:
-            for op in tape.operations:
-                op.queue()
```

Comment:
Now being directly placed into `new_tape`, instead of queued into `new_tape`.

Reply:
Yes queuing is not necessary, you just need a tape with the right operations

Reply:
I just mean to say that if you did something like this:
```python
base_tape = QuantumTape([your_ops])
with fresh_tape:
    tapes, fn = qml.transforms.sign_expand(base_tape)
```
Before this PR, `base_tape.operations` would be re-queued to `fresh_tape`, while this would no longer be true after this PR.

Reply:
I tested my hypothesis and I realize that I'm wrong - it used to queue to the `new_tape`, and that would be queued upward accordingly if using nested tapes. We're not really supporting those anymore so I'm less concerned, but if you do something like what I showed above after your change, you will get an error that you don't get on the current RC branch.

Reply:
Thanks for the details! I don't think users should do that, but that's a possibility indeed.